napari/components/viewer_model.py

```diff
@@ -571,6 +571,99 @@ def add_image(
                 layer_list.append(layer)
             return layer_list
 
+    def add_text(
```

Comment:
this is great, but we're you should add a `view_text` method to `view_layers.py` following the conventions there